.eslintrc.json

```diff
@@ -157,6 +157,37 @@
       "rules": {
         "filenames-simple/naming-convention": "off"
       }
+    },
+    {
+      "files": ["**/api/src/**/*.ts"],
+      "plugins": ["eslint-plugin-jsdoc"],
+      "extends": ["plugin:jsdoc/recommended"],
+      "rules": {
+        "jsdoc/require-jsdoc": [
+          "warn",
+          {
+            "require": {
+              "ArrowFunctionExpression": true,
+              "ClassDeclaration": true,
+              "ClassExpression": true,
+              "FunctionDeclaration": true,
+              "FunctionExpression": true,
+              "MethodDefinition": true
+            },
+            "publicOnly": true
+          }
+        ],
+        "jsdoc/require-description-complete-sentence": "warn",
+        "jsdoc/no-undefined-types": "off",
+        "jsdoc/tag-lines": [
+          "warn",
+          "always",
+          {
+            "startLines": 1,
+            "count": 0
+          }
+        ]
```

Comment:
```suggestion
```
In the spirit of keeping things simple (given that it doesn't affect intellisense and that's what people are likely to be looking at)